f8a_worker/workers/githuber.py

```diff
@@ -87,6 +87,24 @@ def _get_topics(self):
 
         return topics
 
+    def _get_licenses(self):
+        """
+        Get license from GitHub repository through GitHub REST API
+
+        :return: name of repository license
+        """
+        if not self._repo_name:
+            return ""
+
+        # GitHub returns always just one license per repository
+        license = requests.get(
+            "https://api.github.com/repos/{repo_name}/license".format(
```

Comment:
Do we really need the extra API call? I don't know, but this means we will hit the API limit much sooner.

Reply:
Does it count even if I don't use a token for this query? I will try to look whether there not license information inside query which we are doing in this worker.

Reply:
Good point, I think all `api.github.com` calls without authentication are limited to 60 per hour. So this would be definitely no go for us.